roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -36,6 +36,17 @@ server {
   {% if item.value.cache is defined and item.value.cache.enabled | default(false) -%}
   # Fastcgi cache conditions
   set $skip_cache 0;
+
+  # Prevent POST data being held in cache
+  if ($request_method = POST) {
+    set $skip_cache 1;
+  }
+
+  # Prevent common API POST like requests being cached
+  if ($request_method ~ (PUT|PATCH|DELETE)) {
```

Comment:
Wonder if this should be `$request_method ~ ^(PUT|PATCH|DELETE)$` just to be safe?

Reply:
Thanks, long day 😄